src/bootstrap/install.rs

```diff
@@ -49,6 +49,12 @@ pub fn install(build: &Build, stage: u32, host: &str) {
         install_sh(&build, "docs", "rust-docs", stage, host, &prefix,
                    &docdir, &libdir, &mandir, &empty_dir);
     }
+
+    for target in build.config.target.iter().filter(|target| *target != host) {
```

Comment:
Ah yeah that's always true, config.target is a superset of config.host which always contains config.build